### ISSUE TITLE
New Maps

### ISSUE THREAD

**Issue by huntrben** [CIS-499-Group3/Team3-Project#13]
We said we'd have a "new map" when all pellets are eaten. Did you want to actually have new maps drawn, or just reset the map? If you wanted new maps, I'd like to be the one to make them. Mostly cause I'm spending all day every day in spread sheets already

**Comment by DaveWeathers** [comment]
I suppose we could have a few maps and randomize them(or in a set rotation), slightly increasing ghost speed each time...  would make the game easily open ended/"unbeatable".

**Comment by huntrben** [comment]
I'm learning a lot about, with the current ghost behaviors, how I'll have to design the maps to take advantage. Since one ghost is currently seeking out each corner, I can create safe routes that it won't take. With the one that changes directions when it hits a wall, I need to keep in mind that it can cut itself off from entire map sections. Should I try to minimize that or take advantage?

**Comment by alan-boggess** [comment]
The intent with the simpleghost was that so long as we don't code with it specifically in mind, it would be able to reach most points on the map.  I say don't worry about it while you're making maps.

Do whatever you want with the cornersghost.  Just make sure each corner is reachable - I haven't coded any capability for the search to "fail to safe."

**Comment by huntrben** [comment]
Yeah. Cornersghost has only given me one problem so far. I can't make the maps any larger than the current size without breaking something. No real problem except my desire for symmetric maps. But that's not necessary for me

**Comment by alan-boggess** [comment]
The ghost and the A\* search both refer to the dimensions of the map that contains them.  If changing map size breaks something, it oughtn't be in those modules.  What specifically were you getting?  Black screen?  Could you get anything from the console log?

**Comment by huntrben** [comment]
It would freeze when a ghost went into the top right corner. I'll see if I can see what the problem actually was.

**Comment by alan-boggess** [comment]
That is damn odd - I had a similar bug on the default map which turned out to be the path not updating properly.

Is this on a map you've included in an update?  And was it just that corner?

**Comment by huntrben** [comment]
Yeah. just that corner on an older map I made. Like I said, just making the map smaller was a simple fix, so I didn't include it in the version on here
